src/50_Samples_%26_Templates/Poll.html

```diff
@@ -114,25 +103,26 @@ <h4 id="header1">Flightless bird poll</h4>
         <ul>
           [[range $key, $value := .Questions]]
           <li>
+              <input id="[[$key]]" name="answer" value="[[$key]]" type="radio" on="change:poll1.submit">
```

Comment:
Works for me
<img width="1280" alt="screen shot 2017-02-28 at 9 44 52 am" src="https://cloud.githubusercontent.com/assets/820891/23400077/93bbd79e-fd9a-11e6-959b-96f8934a2a5f.png">